Compare-NET-Objects-Tests/IgnoreOrderTests.cs

```diff
@@ -515,6 +515,38 @@ public void GenericEntityListMultipleItemsWithIgnoreCollectionOrderTest()
             Assert.IsFalse(_compare.Compare(genericEntity, genericEntityCopy).AreEqual);
         }
 
+        [Test]
+        public void BaseClassCollectionWithIgnoreCollectionOrderTest()
+        {
+            var list1 = new List<Officer>();
+            var list2 = new List<Officer>();
+
+            var officer1 = new DeriveFromOfficer()
+            {
+                ID = 1,
+                Name = "John",
+                Type = Deck.Engineering,
+                HomeAddress = "Address",
+            };
+
+            var officer2 = new Derive2FromOfficer()
+            {
+                ID = 2,
+                Name = "Jill",
+                Type = Deck.AstroPhysics,
+                Email = "a@a.com",
+            };
+
+            list1.Add(officer1);
+            list1.Add(officer2);
+            list2.Add(officer1);
+            list2.Add(officer2);

```

Comment:
should this test add the officers in a different order for each list?

Reply:
Oops, had it correct and swapped it and made it wrong :( - fixed now, thanks!